src/swift/Block.swift

```diff
@@ -104,8 +104,8 @@ public class DispatchWorkItem {
 /// uses @convention(block). This is to avoid unnecessary bridging between
 /// C blocks and Swift closures, which interferes with dispatch APIs that depend
 /// on the referential identity of a block. Particularly, dispatch_block_create.
-internal typealias _DispatchBlock = @convention(block) () -> Void
-internal typealias dispatch_block_t = @convention(block) () -> Void
+internal typealias _DispatchBlock = @escaping @convention(block) () -> Void
```

Comment:
@slavapestov thanks! just fixed it.

I'm also getting an error now for the block being optional. I submitted a bug, this seemed to be an issue also with `@noescape`, but with other implications now maybe, if those blocks are defaulted to noescaping: [SR-2444](https://bugs.swift.org/browse/SR-2444)